### PR TITLE
feat: oxocarbon

### PR DESCRIPTION
**Pull request by amadalusia**
This PR adds some oxocarbon wallpapers for the pleasure of others, myself included.

The only information I can gather about 01.png is that it was uploaded to alphacoders by someone named "patrika".

02.png has been credited in a previous commit (i think it was "add another wallpaper"?)

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -28,10 +28,11 @@
       # Complete repository: larger and more varied collection.
       # Naturally, means longer build times.
       full = wallpkgs;
-      wallpkgs = callWallpaper null;
```
Comment (NotAShelf) [inline flake.nix:31]: `callWallpaper` is designed to handle null, not an empty string.

Is there a good reason why you're making this change?

Comment (amadalusia) [reply flake.nix:31]: it causes nix flake show and the direnv file to have a panic attack, because it "couldn't" coerce it into a string

Comment (amadalusia) [reply flake.nix:31]: side effects of adding back the null
image

Comment (NotAShelf) [reply flake.nix:31]: I see the problem actually. Setting it to `""` doesn't fix the problem. I'll handle it better in the derivation.

Comment (amadalusia) [reply flake.nix:31]: found and fixed it myself.

**Comment by NotAShelf** [comment]
Cheers.

**Pull request merged**